packages/cozy-harvest-lib/src/components/AccountModal.jsx

```diff
@@ -124,10 +124,10 @@ export class AccountModal extends Component {
               selectedAccount={account}
               accountsAndTriggers={accountsAndTriggers}
               onChange={option => {
-                pushHistory(`/accounts/${option.account._id}`)
+                replaceHistory(`/accounts/${option.account._id}`)
```

Comment:
same here, push is the right thing to do here

Reply:
Updated ✅